action.yml

```diff
@@ -128,6 +128,22 @@ inputs:
   return_full_payload:
     description: 'Return full payload from GitHub rather than a trimmed-down version of the response'
     required: false
+  warning_threshold:
+    description: 'How many days before this issue becomes overdue should the due soon label be added'
+    required: false
+    default: '7'
+  almost_overdue_label:
+    description: 'Optional custom label text for the due soon label'
+    required: false
+    default: 'Due Soon'
+  overdue_label:
+    description: 'Optional custom label text for the overdue label'
+    required: false
+    default: 'Overdue'
+  custom_priority_labels:
```

Comment:
I don't think we need to add support for custom labels at this point

Reply:
Should I at least keep custom priority labels, since teams already use their own priority labels?

Reply:
For now to reduce complexity, let's hard code them and if another consumer needs to use different labels we can add support for it!

Reply:
My team specifically uses different labels, and I was originally creating this helper for our usage 😅

Reply:
But the labels are quite similar so I can talk to my team about changing them to match this helper instead